### PR TITLE
missing images in blogs

### PR DESCRIPTION
**Pull request by rakeshPRaghu**
This PR will add the missing images to the following blogs

1. Inspired by SCale17x — a short history of data storage
2. Running your own DBaaS based on your preferred DBs, Kubernetes operators and containerized storage
3. The Myth of THE database

### CODE REVIEW

File: website/src/blogs/inspired-by-scale17x-e2-80-8a--e2-80-8aa-short-history-of-data-storage.md [hunk 1]
```diff
@@ -42,27 +44,38 @@ Back then, it all sort of made sense. You needed software and systems that could
...
 
 So, what else?
-![https://cdn-images-1.medium.com/max/800/0*VzhXdDLpxtERp1Sk](https://lh5.googleusercontent.com/uIhhtZPwN54Uedbe_dG4TvCXH93sRVcjUXLHTFUPdEPCiQY1PTAbR1YG74riT4ZXM4YkLUh1AB3o6tm0B9mCJhW3IA8t3WLZdOuF6FYSZQY4g8YNV6jd-74Eg6GxoqTipETxU3Jk)
+
+![https://cdn-images-1.medium.com/max/800/0*VzhXdDLpxtERp1Sk](https://cdn-images-1.medium.com/max/800/0*VzhXdDLpxtERp1Sk)
+
 Perhaps most importantly, the workloads — the way DBs are built and run — has changed. They are of much shorter duration, typically much smaller, and they move around! This is not your father’s monolith!
-![https://cdn-images-1.medium.com/max/800/0*Yuk0vAdoZe4qp24s](https://lh3.googleusercontent.com/CDsuKlaIdhRlDvXXmMeuCNQ5hyO587JtKXrBctGqRErHZxvNY06Cx6JbclRlCbx86-fmgeBGRg6_guSNItC8SII58vTgou9xfe_Y31i1eDNEpO-w5RF00WNg_XcuY6Og09QhAXIE)
+
+![https://cdn-images-1.medium.com/max/800/0*Yuk0vAdoZe4qp24s](https://cdn-images-1.medium.com/max/800/0*Yuk0vAdoZe4qp24s)
```
Comment (Pallavi-PH) [inline website/src/blogs/inspired-by-scale17x-e2-80-8a--e2-80-8aa-short-history-of-data-storage.md:59]: Can we change this alt text?

Comment (Pallavi-PH) [reply website/src/blogs/inspired-by-scale17x-e2-80-8a--e2-80-8aa-short-history-of-data-storage.md:59]: Please do the same in other places where URL is added as alt text

File: website/src/blogs/inspired-by-scale17x-e2-80-8a--e2-80-8aa-short-history-of-data-storage.md [hunk 2]
```diff
@@ -31,7 +31,9 @@ Regarding #2 — *where did the idea come from* — when trying to answe
 We put together a few slides that offer a light-hearted approach to answering this question.
 
 **The Era of Slow Disk Drives and Scale-up Architectures**
-![https://cdn-images-1.medium.com/max/800/0*6SPflCuTGC2rWXmH](https://lh3.googleusercontent.com/5sIOx2RN3WtVDqKMxZBCb7KhQRudb4Y7p7SlJ8RmR9JAqw5qLuAQRsoZePf21-e6NkA6_8Eo2oZIudCtqYTICYVj42C10H-Px6NUTaGyQwO1UjEcdXxwaRVdmijtMi7bCL4LuHG3)
+
+![https://cdn-images-1.medium.com/max/800/0*6SPflCuTGC2rWXmH](https://cdn-images-1.medium.com/max/800/0*6SPflCuTGC2rWXmH)
```
Comment (satyapriyamishra222) [inline website/src/blogs/inspired-by-scale17x-e2-80-8a--e2-80-8aa-short-history-of-data-storage.md:35]: Let's try to understand what the image says and accordingly let's set the alt text for that image.

File: website/src/blogs/inspired-by-scale17x-e2-80-8a--e2-80-8aa-short-history-of-data-storage.md [hunk 3]
```diff
@@ -42,27 +44,38 @@ Back then, it all sort of made sense. You needed software and systems that could
 
 **What has changed and how is that changing the data layer?**
 
-![https://cdn-images-1.medium.com/max/800/0*ZJ8CaKQdxEesL7eE](https://lh3.googleusercontent.com/I8EEDuFVhtL2yoa5VUjoOOGvAO5X7o5miPjxV3fe4rombAU9Gz6ZA7sD-C3Alw8fRVpyMzCqW-OqCsGib5RyW0pi-7xJnZpdruVdGKMDeuU-P9sca_2XY6X1itGjMREag0MoLj8D)
+![https://cdn-images-1.medium.com/max/800/0*ZJ8CaKQdxEesL7eE](https://cdn-images-1.medium.com/max/800/0*ZJ8CaKQdxEesL7eE)
```
Comment (satyapriyamishra222) [inline website/src/blogs/inspired-by-scale17x-e2-80-8a--e2-80-8aa-short-history-of-data-storage.md:47]: Let's try to understand what the image says and accordingly let's set the alt text for that image.

File: website/src/blogs/inspired-by-scale17x-e2-80-8a--e2-80-8aa-short-history-of-data-storage.md [hunk 4]
```diff
@@ -42,27 +44,38 @@ Back then, it all sort of made sense. You needed software and systems that could
...
+
+![https://cdn-images-1.medium.com/max/800/0*ZFFma1SdaG9GaiAS](https://cdn-images-1.medium.com/max/800/0*ZFFma1SdaG9GaiAS)
+
 Secondly, instead of having a one size fits all storage system that is limited in its ability to make trade-offs for each of the potentially thousands of workloads, each workload has its own storage system with OpenEBS. The specific needs of those workloads and of the “two pizza” teams that build and run the workloads are each served by their own OpenEBS. This limits the need for cross-functional team meetings in which members argue about how they want the storage to behave. They simply invoke their particular flavor via a Helm chart and, with the help of the right storage classes, away they go!
-![https://cdn-images-1.medium.com/max/800/0*w9bt1flsblASWqcG](https://lh5.googleusercontent.com/WTaj_EcsD5vq_q2EmSxpOByODAp4uOFmX0x8mvCulxvQsG8pGqmpxC0QFk2xnMpCz1qRkkwrP8cAN-7e2PRMmT1cpDFtfUnk2WCptEAbD9xqURIu19rqTxt6Tj4WhVTC-GWrsgoI)
+
+![https://cdn-images-1.medium.com/max/800/0*w9bt1flsblASWqcG](https://cdn-images-1.medium.com/max/800/0*w9bt1flsblASWqcG)
```
Comment (satyapriyamishra222) [inline website/src/blogs/inspired-by-scale17x-e2-80-8a--e2-80-8aa-short-history-of-data-storage.md:77]: There might be some places where the relative path of the image is not correct and that comes as a broken image. Let's look out for those images and then we fix them.

File: website/src/blogs/how-are-tiktok-flipkart-kubesphere-and-others-using-openebs-for-local-volumes.md [hunk 5]
```diff
@@ -11,7 +11,7 @@ excerpt: How are TikTok, Flipkart, KubeSphere, and others using OpenEBS for Loca
 
 We have recently seen a massive increase in the usage of different flavors of OpenEBS Local PV. We estimate by looking at container pulls for underlying components combined with some call home data for those users of OpenEBS that enable the capturing of metrics that the weekly new deployments of OpenEBS for LocalPV increased by nearly 10x during 2020. This can be attributed to the fact that more and more cloud native Stateful applications are moving into Kubernetes
 
-![LocalPv Deployment](https://admin.mayadata.io/content/images/2021/03/Local-PV-Deployment.PNG)
+![LocalPv Deployment](/images/blog/Local-PV-Deployment.png)
```
Comment (satyapriyamishra222) [inline website/src/blogs/how-are-tiktok-flipkart-kubesphere-and-others-using-openebs-for-local-volumes.md:14]: Is this all @rakeshPRaghu  ? Are there no more images that are missing ?

Comment (rakeshPRaghu) [reply website/src/blogs/how-are-tiktok-flipkart-kubesphere-and-others-using-openebs-for-local-volumes.md:14]: Yes @satyapriyamishra222, I have gone through the blogs again only these images were missing.

Comment (Pallavi-PH) [inline website/src/blogs/how-are-tiktok-flipkart-kubesphere-and-others-using-openebs-for-local-volumes.md:14]:
```suggestion
![LocalPv Deployment](/images/blog/local-pv-deployment.png)
```

File: website/src/blogs/inspired-by-scale17x-e2-80-8a--e2-80-8aa-short-history-of-data-storage.md [hunk 6]
```diff
@@ -42,27 +44,38 @@ Back then, it all sort of made sense. You needed software and systems that could
 
 **What has changed and how is that changing the data layer?**
 
-![https://cdn-images-1.medium.com/max/800/0*ZJ8CaKQdxEesL7eE](https://lh3.googleusercontent.com/I8EEDuFVhtL2yoa5VUjoOOGvAO5X7o5miPjxV3fe4rombAU9Gz6ZA7sD-C3Alw8fRVpyMzCqW-OqCsGib5RyW0pi-7xJnZpdruVdGKMDeuU-P9sca_2XY6X1itGjMREag0MoLj8D)
+![Flash and NVMe extrememly fast. Striping slows them down. DAS beats storage](https://cdn-images-1.medium.com/max/800/0*ZJ8CaKQdxEesL7eE)
```
Comment (Pallavi-PH) [inline website/src/blogs/inspired-by-scale17x-e2-80-8a--e2-80-8aa-short-history-of-data-storage.md:47]:
```suggestion
![Flash and NVMe extremely fast. Striping slows them down. DAS beats storage](https://cdn-images-1.medium.com/max/800/0*ZJ8CaKQdxEesL7eE)
```

**Review by Pallavi-PH — APPROVED** [review]
Lgtm

**Review by IsAmrish — APPROVED** [review]

**Review by satyapriyamishra222 — APPROVED** [review]

**Review by AVRahul — APPROVED** [review]

**Pull request merged**